apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -785,6 +780,47 @@ class ScriptEditor extends React.Component {
           </div>
         </CollapsibleEditorSection>
 
+        <CollapsibleEditorSection title="Unit Calendar Settings">
+          <label>
+            Show Calendar
+            <input
+              type="checkbox"
+              checked={this.state.showCalendar}
+              style={styles.checkbox}
+              onChange={() =>
+                this.setState({showCalendar: !this.state.showCalendar})
+              }
+            />
+            <HelpTip>
+              <p>
+                Check to enable the calendar view on the Unit Overview Page. The
+                calendar displays each lesson and generally how long it will
+                take as well how many weeks the unit is expected to take in
+                general. (Actual calendar UI coming soon!)
+              </p>
+            </HelpTip>
+          </label>
+          <label>
+            Instructional Minutes Per Week
+            <HelpTip>
+              <p>
+                Number of instructional minutes to allocate to each week in the
+                calendar. Lessons will be divided across the days/weeks in the
+                calendar based on their length of time.
+              </p>
+            </HelpTip>
+            <input
+              value={this.state.weeklyInstructionalMinutes}
```

Comment:
Currently, you can edit instructional minutes but not check show calendar. Is that the behavior we want? I don't immediately see any issues with it, just raising the question

Reply:
@bethanyaconnor why do you say this?

Reply:
There's protections to make sure someone can't check "show calendar" without adding setting "weekly instructional minutes" but not the opposite. As they feel like an all or nothing set of settings, I was curious whether we should prevent this behavior but I definitely don't feel strongly either way.

Reply:
Ah I see what you are saying now. Making sure you have both or neither seems like a good idea

Reply:
I guess one other scenario I can see here is if you don't want to show calendar for now but you will eventually so you're just filling in instructional minutes

Reply:
Sure check with Tess but I also think this is not a ship blocker so feel free to merge and then update if that works better.

Reply:
+1 -- I just mentioned this as an observation and I definitely don't think it needs to be changed in this PR, if ever